check-hazelcast-health.sh

```diff
@@ -1,15 +1,31 @@
-#!/bin/bash
-
-attempts=0
-max_attempts=30
-until $(curl --silent --fail "127.0.0.1:5701/hazelcast/health/ready"); do
-  if [ ${attempts} -eq ${max_attempts} ];then
-      echo "Hazelcast not responding"
-      cat hz.log
-      ps -aux
-      exit 1
-  fi
-  printf '.'
-  attempts=$(($attempts+1))
-  sleep 2
-done
\ No newline at end of file
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Source the latest version of `abstract-simple-smoke-test.sh` from the `hazelcast-docker` repository and include in current shell
+# TODO Use `master` once merged
+abstract_simple_smoke_test_script_content=$(curl --silent https://raw.githubusercontent.com/hazelcast/hazelcast-docker/DI-215-Add-additional-Deb/RPM/Brew-tests/.github/scripts/abstract-simple-smoke-test.sh)
+# shellcheck source=/dev/null
+. <(echo "${abstract_simple_smoke_test_script_content}")
+
+function get_logs() {
```

Comment:
May be move `get_logs()` to `abstract-simple-smoke-test.sh`?
Also may be rename to `get_hz_logs()`

Reply:
> May be move get_logs() to abstract-simple-smoke-test.sh

It's an abstract class - it doesn't know how to get the logs. Only the children know.

> Also may be rename to get_hz_logs()

Yeah that's fair.

a7d0c673cea659cba1d2d33066ec137f9c100411

Reply:
understood. 
In that case I would have hoped `abstract-simple-smoke-test.sh` implements `get_logs()` and exits unless implemented by caller. Perhaps `abstract-simple-smoke-test.sh` can't be invoked solo
Anyways, just a thought and not saying to change it